crates/task-impls/src/request.rs

```diff
@@ -69,8 +63,6 @@ pub struct NetworkRequestState<TYPES: NodeType, I: NodeImplementation<TYPES>> {
     /// The node's id
     pub id: u64,
     /// A flag indicating that `HotShotEvent::Shutdown` has been received
-    pub shutdown_flag: Arc<AtomicBool>,
-    /// A flag indicating that `HotShotEvent::Shutdown` has been received
     pub spawned_tasks: BTreeMap<TYPES::Time, Vec<JoinHandle<()>>>,
```

Comment:
i dont think we will need this or shutdown flag, im spawning short lived tasks now